tests/changelog.spec.js

```diff
@@ -0,0 +1,55 @@
+import { test, expect } from "@playwright/test";
+const AxeBuilder = require("@axe-core/playwright").default;
+
+// Changelog Page
+test("Changelog has title", async ({ page }) => {
+  await page.goto("/changelog");
+  await expect(page).toHaveTitle("LinkFree user changelog");
+});
+
+// Navigating to Changelog Page
+test("Navigate to Changelog", async ({ page }) => {
+  await page.goto("/roadmap");
+  await page.getByRole("link", { name: "See full list" }).click();
+  await expect(page.locator("h1")).toHaveText("Changelog");
+});
+
+// Changelog Listed
+test.fixme("Events listed", async ({ page }) => {
```

Comment:
Incorrect name, and test will not run as it has `fixme`

```suggestion
test("Changelog listed", async ({ page }) => {
```